.circleci/config.yml

```diff
@@ -45,11 +45,55 @@ jobs:
       - run:
           name: Run smoketests
           command: |
-            docker run -it -e RUNNING_IN_ARTMAN_DOCKER=True --name smoketest artman /bin/bash -c "git clone https://github.com/googleapis/googleapis /tmp/googleapis_smoketest; python3 /artman/test/smoketest_artman.py --root-dir=/tmp/googleapis_smoketest --log=/tmp/smoketest.log" || OUT=$?
-            docker cp smoketest:/tmp/smoketest.log /tmp/smoketest.log
-            if [ $OUT -ne 0 ];then
-              exit $OUT
+            if [ "${RUN_SMOKETEST}" ]; then
```

Comment:
I still don't know why this needs to be flag-guarded, but won't block the review over it.

Reply:
The whole smoketest took ~1 hour, I don't want to have it running during each PR review, but want to have it running automatically when the PR gets merged. I didn't figure out other way except using the circleci project env variable.

Reply:
>  I don't want to have it running during each PR review, but want to have it running automatically when the PR gets merged.

Doesn't a branch filter on `master` work for this?

Reply:
FWIW, the way acceptance tests are set up in Ruby is that they run only if `CIRCLE_BRANCH` == `master` and `CIRCLE_PR_NUMBER` is empty.

Reply:
Probably will work in practice, as the PR is most likely from a non-master forked branch. But PR like this (https://github.com/googleapis/artman/pull/386) will trigger the full smoketest.